easybuild/easyconfigs/b/Blitz++/Blitz++-0.10-foss-2016a.eb

```diff
@@ -0,0 +1,25 @@
+easyblock = 'ConfigureMake'
+
+name = 'Blitz++'
+version = '0.10'
+
+homepage = 'http://blitz.sourceforge.net/'
+description = """Blitz++ is a (LGPLv3+) licensed meta-template library for array manipulation in C++
+ with a speed comparable to Fortran implementations, while preserving an object-oriented interface"""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+sources = ['blitz-%(version)s.tar.gz']
+source_urls = ['https://sourceforge.net/projects/blitz/files/blitz/%(name)s %(version)s',
+               'download']
```

Comment:
@SimonPinches I'm not seeing the easyconfig for goolf-1.5.16 in here?

Also, I stand to my point that this looks wrong.
What you have here now is that you have two separate download URLs listed: `https://sourceforge.net/projects/blitz/files/blitz/%(name)s %(version)s` and `download`.
The latter is obviously wrong.

What you're after is `https://sourceforge.net/projects/blitz/files/blitz/%(name)s %(version)s/<tarball>/download`,
which can be achieved by specifying a list with a single 2-element tuple, i.e.:

```python
source_urls = [('https://sourceforge.net/projects/blitz/files/blitz/%(name)s/%(version)s', 'download')]
```

This tells EasyBuild to grab the 1st element of the tuple, glue the tarball name after it, and then glue the 2nd part of the tuple (`download`) after that...